client/src/app/shared/images/global-icon.component.ts

```diff
@@ -54,7 +54,9 @@ const icons = {
   'users': require('!!raw-loader?!../../../assets/images/global/users.svg').default,
   'search': require('!!raw-loader?!../../../assets/images/global/search.svg').default,
   'refresh': require('!!raw-loader?!../../../assets/images/global/refresh.svg').default,
-  'npm': require('!!raw-loader?!../../../assets/images/global/npm.svg').default
+  'npm': require('!!raw-loader?!../../../assets/images/global/npm.svg').default,
+  'arrows-angle-expand': require('!!raw-loader?!../../../assets/images/global/arrows-angle-expand.svg').default,
```

Comment:
Please check if you can use icons from https://robbiepearce.com/softies/ to keep consistency with the rest of the project

Reply:
Note that softies has fullscreen in/out icons, which we already use and could make good replacements.